content/patterns/button/examples/button_idl.html

```diff
@@ -18,52 +18,52 @@
     <script src="js/button_idl.js"></script>
   </head>
   <body>
-    <nav aria-label="Related Links" class="feedback">
+    <nav aria-label="関連する Issues" class="feedback">
       <ul>
-        <li><a href="https://github.com/w3c/aria-practices/projects/14">Related Issues</a></li>
-        <li><a href="../button-pattern.html">Design Pattern</a></li>
+        <li><a href="https://github.com/w3c/aria-practices/projects/14">関連する Issues</a></li>
+        <li><a href="../button-pattern.html">デザインパターン</a></li>
       </ul>
     </nav>
 
     <main>
-      <h1>Button Examples (IDL Version)</h1>
+      <h1>ボタンの例（IDL バージョン）</h1>
+      <!-- af8f4c1d97ead402022d704754c3d382c4a2cef2 -->
 
       <section>
-        <h2>About This example</h2>
+        <h2>この例について</h2>
         <img alt="" class="example-page-example-icon" src="../../../images/pattern-button.svg">
-        <p>The following examples of the <a href="../button-pattern.html">Button Pattern</a> demonstrate a new JavaScript syntax for coding ARIA attributes.</p>
+        <p><a href="../button-pattern.html">ボタンパターン</a>の以下の例は、ARIA 属性をコーディングするための新しい JavaScript 構文を示しています。</p>
         <p>
-          The JavaScript for the example buttons on this page uses the <a class="specref" href="#idl-interface">IDL Interface defined in ARIA 1.2</a>.
-          The purpose of these examples is to illustrate how to use ARIA Attribute Reflection and provide a test case for browser and assistive technology interoperability.
-          Specifically, the <code>role</code> and <code>ariaPressed</code> attributes are accessed using dot notation instead of <code>setAttribute()</code> and <code>getAttribute()</code>.
-          In all other respects, these examples are identical to the <a href="button.html">Button Examples</a>.
+          このページの例の JavaScript は、<a class="specref" href="#idl-interface">ARIA 1.2 で定義されている IDL インターフェース</a>を使用しています。
+          これらの例の目的は、ARIA 属性リフレクションの使用方法を示し、ブラウザと支援技術の相互運用性のテストケースを提供することです。
```

Comment:
illustrateなので「説明する」ですかね……
```suggestion
          これらの例の目的は、ARIA 属性リフレクションの使用方法を説明し、ブラウザと支援技術の相互運用性のテストケースを提供することです。
```